repository-containers/github.com/tensorflow/addons/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "Tensorflow Addons SIG Dev Container",
+
+	"dockerFile": "Dockerfile",
+	
+    	// Uncomment to enable Nvidia support
+	// "runArgs": ["--runtime=nvidia"],
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": { 
+		"terminal.integrated.shell.linux": null,
+		"python.formatting.provider": "black",
+		"python.testing.pytestEnabled": true,
```

Comment:
@seanpmorgan Let me know if we could enable pytest by default or the discovery it is not working.

Reply:
The integration is commented by default cause vscode-python upstream bugs with pytest:
https://github.com/microsoft/vscode-python/issues/8678
https://github.com/microsoft/vscode-python/issues/9553